Modules/BenchPress.Azure/Public/CustomAssertions/BeInResourceGroup.ps1

```diff
@@ -30,21 +30,31 @@ function ShouldBeInResourceGroup ($ActualValue, [string]$ExpectedValue, [switch]
 
     if ($Negate) { $succeeded = -not $succeeded }
     $failureMessage = "ConfirmResult is null or empty."
-  }
-  else {
-    $resourceGroupName = $ActualValue.ResourceDetails.ResourceGroupName
+  } else {
+    if ($ActualValue.ResourceDetails.ResourceGroupName){
```

Comment:
It looks like the discussion about testing for whether a property exists applies here as well (and in the `elseif`).